### PR TITLE
Portfolio mobile final

### PR DESCRIPTION
**Pull request by dialite**
This request "portfolio-mobile-final branch" contains the about section and the work profile section. All required stylings were implemented to replicate the template 1 of figma content.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,24 +20,165 @@
...
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+              <button type="submit">See Project</button>
+            </div>
+          </section>
+          <!-- About me headline section -->
+          <section class="about-me" class="flex">
+            <div>
```
Comment (mckent05) [inline index.html]: - To follow proper HTML semantics, please it is advisable to have just one `class` property, please fix this 😄 😄

File: style.css [hunk 2]
```diff
@@ -86,3 +96,267 @@ ul {
...
+  margin-bottom: 28px;
+  margin-left: 16px;
+}
+
+button:hover {
+  text-decoration: underline;
+  color: #2230d2;
+  background-color: #d2daff;
+}
+
```
Comment (mckent05) [inline style.css]: - There are some little differences in your button styling when it is hovered on, please confirm the right styling. The color should be `#6070FF`. Please try to fix this for all your buttons 😄 😄

File: style.css [hunk 3]
```diff
@@ -86,3 +96,267 @@ ul {
...
+  background-repeat: no-repeat;
+  background-position: 16px 16px;
+  border-radius: 16px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 16px;
+  width: 327px;
+  height: 552px;
+}
```
Comment (mckent05) [inline style.css:125]: - According to the project requirements, the content of your project cards should be aligned with flexbox. Please try to fix this 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @dialite , 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there


## Highlights
- Linters are passing ✔️ ✔️ 
- Your portfolio is gradually coming together and looking really nice 👏 👏 
- Descriptive commit messages 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
**[OPTIONAL]** -  Please note that it is very important that your website fits nicely across all mobile screens, hence you should prioritize the responsiveness of your site. Currently, your portfolio does not seem to be responsive. I would strongly recommend you look into fixing this 🙏 🙏 , it would help in making your project look even better. You may find this [article](https://www.w3schools.com/css/css3_mediaqueries.asp) useful or you can consider you `%` or `vw` units for your containers width

Your project currently looks like this: 
portfolio mobile
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dialite** [comment]
Changes were just updated to the styles.css linters error

File: index.html [hunk 4]
```diff
@@ -20,24 +20,165 @@
...
+          <li><a class="contacts" href="#"><img src="img/twitter-icon.png" alt='Twitter' /></a></li>
+          <li><a class="contacts" href="#"><img src="img/linkedin-icon.png" alt='linkedIn' /></a></li>
+          <li><a class="contacts" href="#"><img src="img/mail-icon.png" alt="Mail"/></a></li>
+          <li><a class="contacts" href="#"><img src="img/github-icon.png" alt="Github" /></a></li>
+          <li><a class="contacts" href="#"><img src="img/thumbup-icon.png" alt="Thumbup" /></a></li>
+        </ul>
+
+        <!-- Works Section -->
+         <section class="work">
+            <div class="grid-item frame-case-a">
```
Comment (1GORDON) [inline index.html:45]: Great job @dialite 🤝

- [ ] [Optional] As stated by the previous reviewer, kindly make sure the contents of the individual cards are aligned with `display: flex`.

File: index.html [hunk 5]
```diff
@@ -20,24 +20,165 @@
...
+              <!-- <img src="img/Snapshoot-Portfolio-a.png" alt=""> -->
```
Comment (1GORDON) [inline index.html:46]: Great job @dialite 🤝

- [ ] [Optional] Kindly remove commented code from your project as this is regarded as bad practice.

File: index.html [hunk 6]
```diff
@@ -20,24 +20,165 @@
...
+              <p>
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="flex tool-content">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+              <button type="submit">See Project</button>
```
Comment (1GORDON) [inline index.html:108]: Great job @dialite 🤝

- [ ] [Optional] As stated by the previous reviewer, kindly implement the button interactions of `hover`, and `active` across all buttons correctly.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @dialite ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (dialite) [reply index.html:108]: Thank you so much. I will act accordingly

**Pull request merged**